### PR TITLE
add trivy scan to docker

### PR DESCRIPTION
**Pull request by farodin91**
-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

**Comment by farodin91** [comment]
@porunov This adds the last missing part from the janusgraph-docker repo.

**Review by porunov — APPROVED** [review]
LGTM. Thank you @farodin91 !

**Comment by porunov** [comment]
@farodin91 looks like it fails with `UserError: Path does not exist: trivy-results.sarif`

**Comment by farodin91** [comment]
@porunov Thank you. It should be fixed now.

**Comment by porunov** [comment]
> @porunov Thank you. It should be fixed now.

Still the same error. Do you know if this PR is a blocker for the release?

**Comment by farodin91** [comment]
It's a release blocker.

File: .github/workflows/ci-release.yml [hunk 1]
```diff
@@ -68,8 +68,10 @@ jobs:
         include:
           - args: ""
             java: 8
+            tag_suffix: "-java-11"
```
Comment (porunov) [inline .github/workflows/ci-release.yml:71]: Shouldn’t this be “-java-8” instead of “-java-11”?

Comment (farodin91) [reply .github/workflows/ci-release.yml]: ups

**Comment by porunov** [comment]
@farodin91 Looks like all the tests passed. However, the required job in JanusGraph settings is set for `dist-tests (-Pjava-11, 11)`, butin this PR the name changes to `dist-tests (-Pjava-11, 11, -java-11)` because of `tag_suffix`. To merge this PR we need temporary disable that setting in JanusGraph Settings > Branches -> `master`. After that we need to re-base all current opened PRs and then enable that branch protection again.

**Review by porunov — APPROVED** [review]

**Comment by porunov** [comment]
Disabled `dist-tests (-Pjava-11, 11)` required check

**Comment by farodin91** [comment]
@porunov Thank you for merging.

**Comment by porunov** [comment]
> @porunov Thank you for merging.

Thank you for the Docker integration!

**Comment by janusgraph-automations** [comment]
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|v0.6|An unhandled error occurred. Please see the logs for details|

### Manual backport
To create the backport manually run:
```
backport --pr 4056
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/JanusGraph/janusgraph/actions/runs/6589858016) for details

**Pull request merged**